mocktarget/errors.json

```diff
@@ -19,15 +19,7 @@
                     },
                     "uuid": "e81e56ff-7f65-4279-817f-dd607585b9e0",
                     "timestamp": "2019-10-11T12:46:25.595Z",
-                    "severity": "error",
-                    "http_context": {
```

Comment:
Should be `"http_context":null`, right?

Reply:
it works in both cases, with a present key and null value and with the key entirely missing. We just need to make sure it is properly documented as optional